ctwrap/__init__.py

```diff
@@ -5,6 +5,5 @@
 from .simulation import *
 from .parser import *
```

Comment:
Please replace `*` by the imported objects (wildcard imports lead to complaints by a delinter)